extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -340,10 +390,38 @@ export class SystemLayer {
    * @private
    */
   onMutedStateUpdate_(isMuted) {
+    
     this.vsync_.mutate(() => {
-      isMuted ?
-        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, '') :
-        this.getShadowRoot().removeAttribute(AUDIO_MUTED_ATTRIBUTE);
+      if(isMuted) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'muted');
+       } else if(this.storeService_.get(StateProperty.PAGE_HAS_AUDIO_STATE)) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'soundon');
```

Comment:
nit: prefer `audioon` and `noaudio`, to match the `noaudio` attribute used by `amp-video`.